Tasks/JavaToolInstaller/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "java-tool-installer",
+    "version": "0.1.0",
+    "description": "Java Tool Installer",
+    "main": "javaToolInstaller.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/microsoft/vsts-tasks.git"
+    },
+    "keywords": [
+        "VSTS",
+        "Tasks",
+        "Java"
+    ],
+    "author": "Microsoft Corporation",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/microsoft/vsts-tasks/issues"
+    },
+    "homepage": "https://github.com/microsoft/vsts-tasks#readme",
+    "dependencies": {
+        "vsts-task-lib": "2.0.1-preview"
```

Comment:
Use latest released version unless you need something that is only available in preview